README.md

```diff
@@ -254,6 +254,7 @@ When learning CS there are some useful sites you must know to get always informe
 - [Unsupervised Sentiment Neuron](https://blog.openai.com/unsupervised-sentiment-neuron/) : Unsupervised Sentiment Neuron 
 - [Robots that learn](https://blog.openai.com/robots-that-learn/) : Robots that Learn 
 - [grakn.ai](https://grakn.ai) : The Database for AI 
+- [fast.ai](http://course.fast.ai/) : Extremely well done course on AI. Takes 7 weeks from zero to hero.
```

Comment:
Great link! 
Here is a suggestion for the description for this link. 
`Free practical *deep learning* for coders without grad-level maths!`